lib/tasks/cap_cutover.rake

```diff
@@ -102,363 +33,13 @@ namespace :cap_cutover do
         preferred_middle_name: row[:preferred_middle_name],
         california_physician_license: (row[:ca_license_number])
       )
-
-      GC.start if total_running_count % 5000 == 0
       if total_running_count % 5000 == 0
+        GC.start
         Rails.logger.debug "#{total_running_count} in #{distance_of_time_in_words_to_now(start_time, true)}"
       end
     end
   end
 
-  desc 'create publication and contribution records from full cap dump'
-  task :create_pubs_from_cap, [:file_location] => :environment do |_t, args|
-    include ActionView::Helpers::DateHelper
-    start_time = Time.zone.now
-    total_running_count = 0
-    @cap_import_logger = Logger.new(Rails.root.join('log', 'cap_create_pmid_pubs.log'))
-    @cap_import_logger.info 'Started cap build pub process ' + Time.zone.now.to_s
-
-    lines = CSV.foreach(args.file_location, headers: true, header_converters: :symbol) do |row|
-      total_running_count += 1
-
-      author = Author.where(cap_profile_id: row[:profile_id]).first
-      pub = Publication.where(pmid: row[:pubmed_id]).first_or_create
-      pub.contributions.where(author_id: author.id).first_or_create(
-        cap_profile_id: row[:profile_id],
-        status: row[:authorship_status],
-        visibility: row[:visibility],
-        featured: row[:featured])
-
-      GC.start if total_running_count % 5000 == 0
-
-      if total_running_count % 1000 == 0
-        Rails.logger.debug "#{total_running_count} in #{distance_of_time_in_words_to_now(start_time, true)}"
-        @cap_import_logger.info total_running_count.to_s + ' in ' + distance_of_time_in_words_to_now(start_time, true)
-      end
-    end
-    @cap_import_logger.info 'Finished import.' + Time.zone.now.to_s
-    @cap_import_logger.info lines.to_s + ' lines of file: ' + args.file_location.to_s + ' were processed.'
-    @cap_import_logger.info total_running_count.to_s + 'records were processed in ' + distance_of_time_in_words_to_now(start_time, true)
-  end
```

Comment:
OK, `create_pubs_from_cap` noted for deprecation in https://github.com/sul-dlss/sul_pub/issues/356#issuecomment-341251365